generate/generate.go

```diff
@@ -21,7 +21,14 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec       *rspec.Spec
+	ExportOpts *ExportOptions
```

Comment:
On Fri, Jul 22, 2016 at 11:20:20AM -0700, Antonio Murdaca wrote:
> > @@ -21,7 +21,14 @@ var (
> >  
> >  // Generator represents a generator for a container spec.
> >  type Generator struct {
> > -	spec *rspec.Spec
> > +	spec       *rspec.Spec
> > +	ExportOpts *ExportOptions
>
> if, of course, it isn't I'm all for moving that to `Save*` but
> afaict it's a generator option

Can you unpack “generator option” for me?  Since #131 we've been
trying to maintain two APIs: a library API and a command line API.  To
me, that means we develop the library API with an eye to expression
and flexibility, and then bind on some (probably less flexible)
convenient wrapper for the command line API.

I can imagine someone wanting to save both a full config and also only
the seccomp part, and don't want those calls competing for control of
the same Generator.ExportOpts resource.